raiden_contracts/data/source/services/ServiceRegistry.sol

```diff
@@ -280,12 +294,17 @@ contract ServiceRegistry is Utils, ServiceRegistryConfigurableParameters {
     /// @notice Sets the URL used to access a service via HTTP.
     /// Only a currently registered service can call this successfully.
     /// @param new_url The new URL string to be stored.
-    function setURL(string memory new_url) public returns (bool _success)  {
+    function setURL(string memory new_url) public returns (bool _success) {
         require(now < service_valid_till[msg.sender], "registration expired");
         require(bytes(new_url).length != 0, "new url is empty string");
         urls[msg.sender] = new_url;
         return true;
     }
+
+    /// A getter function for seeing the length of ever_made_deposits array.
+    function ever_made_deposits_len() public view returns (uint256 _len) {
```

Comment:
It's really weird that solidity does not provide this for public arrays itself.

Reply:
I agree.  It's weird.